lib/livebook/hubs/team_client.ex

```diff
@@ -178,13 +178,14 @@ defmodule Livebook.Hubs.TeamClient do
     %{state | secrets: Enum.reject(state.secrets, &(&1.name == secret.name))}
   end
 
-  defp build_secret(state, %{name: name, value: value}) do
+  defp build_secret(state, %{name: name, value: value, deployment_group_id: deployment_group_id}) do
```

Comment:
I don't think we can assume this key will be there. So instead of passing it as an empty string, what if we do:

```suggestion
  defp build_secret(state, %{name: name, value: value} = attrs) do
```

And then below we do: `deployment_group_id: attrs[:deployment_group_id]`?